contracts/deploy/L2AddWormholeDomainSpell.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.8.9;
+
+interface DaiLike {
+  function rely(address usr) external;
+}
+
+interface WormholeBridgeLike {
+  function file(
+    bytes32 what,
+    bytes32 domain,
+    uint256 data
+  ) external;
+}
+
+contract L2AddWormholeDomainSpell {
+  DaiLike public immutable dai;
+  WormholeBridgeLike public immutable wormholeBridge;
+  bytes32 public immutable masterDomain;
+
+  constructor(
+    DaiLike _dai,
+    WormholeBridgeLike _wormholeBridge,
+    bytes32 _masterDomain
+  ) {
+    dai = _dai;
+    wormholeBridge = _wormholeBridge;
+    masterDomain = _masterDomain;
+  }
+
+  function execute() external {
+    dai.rely(address(wormholeBridge));
```

Comment:
We could add a comment that this is required for the L2 wormhole bridge to be able to burn L2 Dai